api/src/org/apache/cloudstack/api/command/user/template/CreateTemplateCmd.java

```diff
@@ -293,7 +293,7 @@ public void create() throws ResourceAllocationException {
     @Override
     public void execute() {
         CallContext.current().setEventDetails(
-            "Template Id: " + getEntityId() + ((getSnapshotId() == null) ? " from volume Id: " + getVolumeId() : " from snapshot Id: " + getSnapshotId()));
+            "Template Id: " + getEntityUuid() + ((getSnapshotId() == null) ? " from volume Id: " + this._uuidMgr.getUuid(Volume.class,getVolumeId()) : " from snapshot Id: " + this._uuidMgr.getUuid(VirtualMachineTemplate.class,getSnapshotId())));
         VirtualMachineTemplate template = null;
```

Comment:
Should be `Snapshot.class`, can you also please check if other entity types match correctly?